core/api.go

```diff
@@ -158,6 +158,22 @@ type IEntryEvent interface {
 	Uuid() *string
 }
 
+// IItemEvent is IList, ISet and IQueue events common contract.
+type IItemEvent interface {
```

Comment:
This is the copy of `protocol/ItemEvent`. Why duplicates?

Reply:
What do you mean by copy? `protocol/ItemEvent` implements `IItemEvent interface`

Reply:
my mistake. 👍